hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOmMetadataManager.java

```diff
@@ -558,6 +561,128 @@ public void testListKeysWithFewDeleteEntriesInCache() throws Exception {
 
   }
 
+  @Test
+  public void testListOpenFilesFSO() throws Exception {
+    testListOpenFiles(BucketLayout.FILE_SYSTEM_OPTIMIZED);
+  }
+
+  @Test
+  public void testListOpenFilesOBS() throws Exception {
+    testListOpenFiles(BucketLayout.OBJECT_STORE);
+  }
+
+  @Test
+  public void testListOpenFilesLegacy() throws Exception {
+    // OBS and LEGACY should share the same internal structure for the most part
+    // still, testing both here for the sake of completeness
+    testListOpenFiles(BucketLayout.LEGACY);
+  }
+
+  /**
+   * Tests inner impl of listOpenFiles with different bucket types with and
+   * without pagination. NOTE: This UT does NOT test hsync in this since hsync
+   * status check is done purely on the client side.
+   * @param bucketLayout BucketLayout
+   */
+  public void testListOpenFiles(BucketLayout bucketLayout) throws Exception {
```

Comment:
Let's use `@ParameterizedTest` for the same.

```suggestion
  /**
   * Tests inner impl of listOpenFiles with different bucket types with and
   * without pagination. NOTE: This UT does NOT test hsync in this since hsync
   * status check is done purely on the client side.
   */
  @ParameterizedTest
  @EnumSource
  void testListOpenFiles(BucketLayout bucketLayout) throws Exception {
```

(additional import needed for `EnumSource`)